File: csharp/high-scores/HighScores.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class HighScores
{
    List<int> mlstIntHighScores;
    public HighScores(List<int> list) => mlstIntHighScores = list;

    public List<int> Scores() => mlstIntHighScores;

    public int Latest() => mlstIntHighScores.Last();

    public int PersonalBest() => mlstIntHighScores.Max();

    public List<int> PersonalTopThree() => mlstIntHighScores.OrderByDescending(i => i).Take(3).ToList();
}
//exercism submit C:\Users\eric\Exercism\csharp\high-scores\HighScores.cs

File: python/high-scores/high_scores.py
def latest(scores: list) -> int:
    return scores[-1] if scores else None


def personal_best(scores: list) -> int:
    return max(scores) if scores else None


def personal_top_three(scores: list) -> list:
    return sorted(scores, reverse=True)[:3]

# exercism submit  C:\Users\eric\Exercism\python\high-scores\high_scores.py
